pkg/controller/backup_controller.go

```diff
@@ -564,12 +564,15 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 		if c.volumeSnapshotContentLister != nil {
 			// Since VolumeSnapshotContent objects are not currently labeled, get them by using binding from the VolumeSnapshot
 			for _, vs := range volumeSnapshots {
-				vsc, err := c.volumeSnapshotContentLister.Get(*vs.Status.BoundVolumeSnapshotContentName)
-				if err != nil {
-					backupLog.Error(err)
-					continue
+				// nil check just in case the snapshot and the content object did not get bound before returning from the plugins
+				if vs.Status.BoundVolumeSnapshotContentName != nil {
```

Comment:
technically `Status` could be nil too, let's check both just in case:

`if vs.Status != nil && vs.Status.BoundVolumeSnapshotContentName != nil {`